frontend/packages/kubevirt-plugin/src/plugin.ts

```diff
@@ -0,0 +1,60 @@
+import * as _ from 'lodash-es';
+import { Plugin, ResourceNSNavItem, ResourceListPage, ResourceDetailPage, ModelFeatureFlag, YamlTemplate, ModelDefinition } from '@console/plugin-sdk';
+
+import * as models from './models';
+import { vmYamlTemplate } from './yaml-templates';
+
+type ConsumedExtensions = ResourceNSNavItem | ResourceListPage | ResourceDetailPage | ModelFeatureFlag | YamlTemplate | ModelDefinition;
+
+const FLAG_KUBEVIRT = 'KUBEVIRT';
+
+const plugin: Plugin<ConsumedExtensions> = [
+  {
+    type: 'ModelDefinition',
+    properties: {
+      models: _.values(models),
+    },
+  },
+  {
+    type: 'FeatureFlag/Model',
+    properties: {
+      model: models.VirtualMachineModel,
+      flag: FLAG_KUBEVIRT,
+    },
+  },
+  {
+    type: 'NavItem/ResourceNS',
+    properties: {
+      section: 'Workloads',
+      componentProps: {
+        name: 'Virtual Machines',
+        resource: models.VirtualMachineModel.plural,
+        required: FLAG_KUBEVIRT,
+      },
+      mergeAfter: 'Pods', // rendered name of the resource navigation link in the left-side menu
+    },
+  },
+  {
+    type: 'ResourcePage/List',
+    properties: {
+      model: models.VirtualMachineModel,
+      loader: () => import('./components/vm' /* webpackChunkName: "virtual-machines" */).then(m => m.VirtualMachinesPage),
```

Comment:
For consistency with Metal3 plugin, I'd suggest using a `kubevirt-` prefix in the chunk name, e.g. `kubevirt-virtual-machines` or `kubevirt-virtualmachines`.

It will give us better traceability when analyzing webpack-generated chunks that might come from different Console plugins.

Reply:
done